bin/init.sh

```diff
@@ -100,8 +129,10 @@ if [ ! -f /usr/local/bin/helm -a ! -f ${ISTIO_OUT}/helm ] ; then
         rm -rf helm.tgz linux-amd64
 fi
 
+# TODO(nmittler): Remove once tests no longer use the envoy binary directly.
```

Comment:
I would remove the TODO - we do want more tests to use the envoy binary directly (faster, 
matches rawvm env etc)!